tests/Feature/AssignUserToOrganizationsTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature;
+
+use App\Models\Organization;
+use App\Models\User;
+use App\Services\AssignUserToOrganizationsService;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Facades\Http;
+use Tests\TestCase;
+
+class AssignUserToOrganizationsTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function testAssignFunctionWithValidResponse(): void
+    {
+        $user = User::factory()->create();
```

Comment:
Maybe move creatig user to `setUp` method, u are doing same thing many times.

Reply:
moved creating user and service to setup